app/script/entity/Conversation.js

```diff
@@ -0,0 +1,628 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.entity = z.entity || {};
+
+
+z.entity.Conversation = class Conversation {
+  /**
+   * Constructs a new conversation entity.
+   * @class z.entity.Conversation
+   * @param {string} conversation_id - Conversation ID
+   */
+  constructor(conversation_id = '') {
+    this.id = conversation_id;
+    this.creator = undefined;
+    this.type = ko.observable();
+    this.name = ko.observable();
+    this.input = ko.observable(z.util.StorageUtil.get_value(`${z.storage.StorageKey.CONVERSATION.INPUT}|${this.id}`) || '');
+    this.input.subscribe((text) => {
+      return z.util.StorageUtil.set_value(`${z.storage.StorageKey.CONVERSATION.INPUT}|${this.id}`, text);
```

Comment:
It extended our character limit per line:

![unbenannt](https://cloud.githubusercontent.com/assets/469989/25842148/45e154d2-34a3-11e7-8721-b0e7b3b9bc7e.PNG)